src/main/cpp/multiregion/TravelModel.h

```diff
@@ -101,6 +111,8 @@ class RegionTravel final
 private:
 	RegionId region_id;
 	std::string region_population_path;
+	std::string region_geodistribution_profile_path;
```

Comment:
I think they just need to exist at the sub-simulation level (different regions might need populations generated from different geodistribution profiles, etc).

They can probably be pulled up into the `SingleSimulationConfig` class? I wasn’t sure what level that class and `TravelModel` operate on, but I saw this bit of code in `SimulationConfig.h`, and figured that, if `TravelModel` is where *one* population generation config file path goes, it’s not a bad place for the related files to go, either:

```cpp
/// Gets a path to the population file for this simulation.
std::string GetPopulationPath() const { return travel_model->GetRegionPopulationPath(); };
```